tokio/src/runtime/builder.rs

```diff
@@ -781,6 +788,37 @@ impl Builder {
             self.unhandled_panic = behavior;
             self
         }
+
+        /// Disables the LIFO task scheduler heuristic.
+        ///
+        /// The multi-threaded scheduler includes a heuristic for optimizing
+        /// message-passing patterns. This heuristic results in the **last**
+        /// scheduled task being polled first.
+        ///
+        /// To implement this heuristic, each worker thread has a slot which
+        /// holds the task that should be polled next. However, this slot cannot
+        /// be stolen by other worker threads, which can result in lower total
+        /// throughput when tasks tend to have longer poll times.
+        ///
+        /// This configuration option will disable this heuristic resulting in
+        /// all scheduled tasks being pushed into the worker-local queue, which
+        /// is stealable.
```

Comment:
might be worth stating somewhere that this option does nothing with the single-threaded scheduler?

Reply:
In theory, it should though 😆

Reply:
ah, then maybe we should change this paragraph, which only mentioned the multi-threaded scheduler? https://github.com/tokio-rs/tokio/blob/97f6693cc1c759900c03d6e5205278bb57ea2da7/tokio/src/runtime/builder.rs#L794-L797 and/or add an example disabling the lifo slot in a single threaded scheduler?